src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/EnergyConnector.java

```diff
@@ -28,10 +28,13 @@
  *
  */
 public class EnergyConnector extends SimpleSlimefunItem<BlockUseHandler> implements EnergyNetComponent {
+    public static int RANGE;
 
     @ParametersAreNonnullByDefault
-    public EnergyConnector(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
+    public EnergyConnector(ItemGroup itemGroup, int range, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput) {
```

Comment:
right now this is a breaking change
what i would do is, making another constructor that would make the int range optional

Reply:
can you give me a example? i never heard something like 2nd constructor

Reply:
```java
public EnergyConnector(ItemGroup itemGroup, SlimefunItemStack item, RecipeType recipeType, ItemStack[] recipe, ItemStack recipeOutput, int range = 100) {
	EnergyConnector(itemGroup, item, recipeType, recipe, recipeOutput);
        // And also set range to 'range' variable
}```

Reply:
Using a constructor to assign a public static field is really sketchy. How can you be sure someone doesn't use it before you assign it? Also, what if there are two `EnergyConnector` tiers with different ranges? Then this code totally falls apart...

Can you make it work by the `CargoNet` and `EnergyNet` using the cfg too?